types/setting.go

```diff
@@ -815,6 +821,26 @@ var (
 		ReadOnly: false,
 		Default:  CniNetworkNone,
 	}
+
+	SettingDefinitionRecurringSuccessfulJobsHistoryLimit = SettingDefinition{
+		DisplayName: "Cronjob Successful Jobs History Limit",
+		Description: "This setting specifies how much successful backup or snapshot job history should be kept.",
+		Category:    SettingCategoryBackup,
+		Type:        SettingTypeInt,
+		Required:    false,
+		ReadOnly:    false,
+		Default:     "1",
+	}
+
+	SettingDefinitionRecurringFailedJobsHistoryLimit = SettingDefinition{
+		DisplayName: "Cronjob Failed Jobs History Limit",
+		Description: "\nThis setting specifies how much failed backup or snapshot job history should be kept.",
```

Comment:
Why there was a `/n` at the beginning?
```suggestion
		Description: "This setting specifies how many failed backup or snapshot job histories should be retained.",
```

Reply:
Thank you, I will follow the comment to fix.
https://github.com/longhorn/longhorn-manager/blob/de00e06ea1fcd510afee72cb8132231513953f84/types/setting.go#L837